scripts/zones/Upper_Delkfutts_Tower/mobs/Mimas.lua

```diff
@@ -0,0 +1,28 @@
+-----------------------------------
+-- Area: Upper Delkfutt's Tower
+--  MOB: Mimas
+-----------------------------------
+
+require("scripts/globals/status");
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_SCRIPTED_2HOUR, 1);
```

Comment:
Unneeded, and that mobmod will be deleted soon™

when we call an ability with `useMobAbility()` the skillcheck is ignored and that ability just goes.

Reply:
i added it in case core would ever pick up 2hr support, but looking at wren's pr it probably won't